app/uk/gov/hmrc/nisp/views/account_cope.scala.html

```diff
@@ -22,14 +22,13 @@
 @import uk.gov.hmrc.nisp.controllers.auth.NispUser
 @import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
 
-
 @(nino: String, forecastAmount: BigDecimal, copeEstimate: BigDecimal, totalPension: BigDecimal, authenticationProvider: String,
```

Comment:
Can you remove `forecastAmount` and `totalPension` from the parameters, and all the calling code as they are redudant? Just a bit of tidying up since we've removed them.